src/Rules/Generators/GhcSplit.hs

```diff
@@ -0,0 +1,25 @@
+module Rules.Generators.GhcSplit (generateGhcSplit) where
+
+import Base
+import Expression
+import Oracles
+import Settings.User
+
+generateGhcSplit :: Expr String
+generateGhcSplit = do
+    let yesNo = lift . fmap (\x -> if x then "YES" else "NO")
+    perl <- getBuilderPath Perl
+    let script = "driver" -/- "split" -/- "ghc-split.prl"
```

Comment:
In the rest of the codebase we simply write `"driver/split/ghc-split.prl"`. We use forward slashes everywhere, even on Windows, so I don't see a benefit using `-/-` here.